index.js

```diff
@@ -98,9 +98,7 @@ const ORDERED_DEFAULTS = {
       'babel'
     ],
     module: [
-      'resolve',
```

Comment:
We should not resolve or commonjs modules, they should **never** be included. This is one of the big things that I have seen with our dependencies, where two of the same dependency are included.

Reply:
I guess this is the change I worry most about. But I can't think of any places where we need these off the top of my head.

Reply:
This will be a major version as well, so it won't be forced on anything.